src/@types/event.ts

```diff
@@ -55,6 +55,7 @@ export enum EventType {
     CallReplaces = "m.call.replaces",
     CallAssertedIdentity = "m.call.asserted_identity",
     CallAssertedIdentityPrefix = "org.matrix.call.asserted_identity",
+    CallEncryptionPrefix = "io.element.call.encryption_key",
```

Comment:
Btw, why are these called 'prefix'? They're not prefixes, surely? They're just normal event type names.

Reply:
So this is somewhat up to discussion - I wrote this with the assumption that we would end up making this `m.` prefixed one day, so that's why I called it prefix

Reply:
Oh I see - that kind of prefix. That makes sense :)

Reply:
Btw, `Prefixed` would probably convey this message a bit better, but no need to change them all now.

Reply:
FWIW, I followed the convention used by `CallAssertedIdentityPrefix` for example but yeah agreed